lib/src/endpoints/me.dart

```diff
@@ -155,9 +155,41 @@ class Me extends EndpointPaging {
     return _getPages('$_path/shows', (json) => Show.fromJson(json['show']));
   }
 
+  /// Save shows for the current user. It requires the `user-library-modify`
+  /// scope.
+  /// [ids] - the ids of the shows to save
+  Future<void> saveShows(List<String> ids) async {
+    assert(ids.isNotEmpty, 'No show ids were provided for saving');
+    await _api._put('$_path/shows?' + _buildQuery({'ids': ids.join(',')}));
+  }
+
+  /// Removes shows for the current user. It requires the `user-library-modify`
+  /// scope.
+  /// [ids] - the ids of the shows to remove
+  /// [market] - An ISO 3166-1 alpha-2 country code. If a country code is
+  /// specified, only content that is available in that market will be returned.
+  Future<void> removeShows(List<String> ids, [String market = '']) async {
+    assert(ids.isNotEmpty, 'No show ids were provided for removing');
+    var query = _buildQuery({'ids': ids.join(','), 'market': market});
+    await _api._delete('$_path/shows?' + query);
+  }
+
+  /// Check if passed albums (ids) are saved by current user. The output
+  /// [ids] the ist of id's to check
```

Comment:
The comments on this method seem to be a bit jumbled :)

Reply:
indeed 😅

Reply:
Make it a "list" in line 178 and I'm happy 😆

Reply:
well this is what happens when you code late at night 😆

Reply:
It happens 😄